sw/device/lib/dif/dif_i2c.c

```diff
@@ -269,6 +269,33 @@ dif_result_t dif_i2c_host_set_enabled(const dif_i2c_t *i2c,
 
   uint32_t reg = mmio_region_read32(i2c->base_addr, I2C_CTRL_REG_OFFSET);
   reg = bitfield_bit32_write(reg, I2C_CTRL_ENABLEHOST_BIT, flag);
+  // FIXME #14931 This dif can't set I2C's line loopback
+  mmio_region_write32(i2c->base_addr, I2C_CTRL_REG_OFFSET, reg);
+
+  return kDifOk;
+}
+
+dif_result_t dif_i2c_device_set_enabled(const dif_i2c_t *i2c,
+                                      dif_toggle_t state) {
```

Comment:
nit: Looks like something went awry with spacing here.

Reply:
I'm going to rely on clang fmt to help me get this right, sorry not to have run it on the draft